cmd/harmony/main.go

```diff
@@ -175,6 +177,39 @@ func createGlobalConfig() *nodeconfig.ConfigType {
 	return nodeConfig
 }
 
+func createArchivalNode() (*node.Node, *nodeconfig.ConfigType) { //Mix of setUpConsensusAndNode and createGlobalConfig
+	var err error
+	nodeConfig := nodeconfig.GetGlobalConfig()
+
+	// Currently we hardcode only one shard.
+	nodeConfig.ShardIDString = "10000" //ShardID can be highest positive number of shard.
```

Comment:
what's this about?  ArchivalNode will need to join beacon chain to start.

Reply:
Done.